cache/memcached_client.go

```diff
@@ -244,12 +246,27 @@ func (c *memcachedClient) SetAsync(ctx context.Context, key string, value []byte
 	})
 }
 
-func (c *memcachedClient) GetMulti(ctx context.Context, keys []string) map[string][]byte {
+func toMemcacheOptions(opts ...Option) []memcache.Option {
+	base := &Options{}
```

Comment:
[nit] What if we return `nil` on no `opts`? We would avoid `&Options{}` allocation when no options are provided.